package.json

```diff
@@ -27,7 +29,8 @@
     "lint-fix": "eslint . --fix",
     "test": "cross-env NODE_ENV=test ava",
     "coverage": "cross-env NODE_ENV=test nyc ava",
-    "report-coverage": "nyc report --reporter=text-lcov | coveralls"
+    "report-coverage": "nyc report --reporter=text-lcov | coveralls",
+    "version": "babel-node build/version.js"
```

Comment:
Might want to use `postversion` for this, so that `build/version.js` isn't using the old version when creating the changelog file.